tests/purs/failing/VisibleTypeApplications1.out

```diff
@@ -0,0 +1,20 @@
+Error found:
+in module [33mMain[0m
+at tests/purs/failing/VisibleTypeApplications1.purs:7:11 - 7:18 (line 7, column 11 - line 7, column 18)
+
+  An expression of polymorphic type
+  with the invisible type variable [33ma[0m:
+  [33m                  [0m
+  [33m  forall a. a -> a[0m
+  [33m                  [0m
+  cannot be applied to:
+  [33m     [0m
+  [33m  Int[0m
+  [33m     [0m
```

Comment:
This error can be expanded further by suggesting that the user can change the signature of the inferred type to allow it to be used in type application.

I think that for polymorphic types, it's the case that they're often written out by the user such that we can suggest editing the source, but I feel we can clarify this in the error index for the documentation page.